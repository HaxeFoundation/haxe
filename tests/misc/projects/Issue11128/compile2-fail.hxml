--main Main

# Error cause of reserved name
--custom-target js
