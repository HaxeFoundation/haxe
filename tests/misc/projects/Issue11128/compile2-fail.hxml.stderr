Error: : --custom-target cannot use reserved name js.